docs/index.md

```diff
@@ -146,6 +146,7 @@ limitations under the License.
   * [Lens](./interpreter/lens.html)
   * [Livy](./interpreter/livy.html)
   * [markdown](./interpreter/markdown.html)
+  * [Neo4j](./interpreter/neo4j.html)
```

Comment:
We need to add the index into https://github.com/apache/zeppelin/blob/master/docs/_includes/themes/zeppelin/_navigation.html#L138 as well

Otherwise, it will not appear in navbar